invenio_app_rdm/records_ui/views/decorators.py

```diff
@@ -79,6 +79,26 @@ def view(**kwargs):
     return view
 
 
+def pass_record_from_pid(f):
+    """Decorate a view to pass the latest version of a record."""
+    @wraps(f)
+    def view(**kwargs):
+        pid_type = kwargs.get('pid_type')
+        pid_value = kwargs.get('pid_value')
+        pid_client = kwargs.get('client')
```

Comment:
Where is the ``client`` coming from? I don't see it in the URL pattern?

Reply:
you are right... it was meant for multi-client, and should be something like `/doi/provider.1234-abdce?client=zenodo`... However, for that I would need to access `requests.args`, which is not super clean. Any other ideas?

Reply:
The PID resolution i think should be independent of the provider.